### ISSUE TITLE
Bounding box loss 

### ISSUE THREAD

**Issue by LjIA26** [open-mmlab/mmpose#1950]
Hello, 

I would like to calculate the bounding box loss. But I can't find documentation of how the bounding box is used within HRNET. Could you shed some light in the matter?

**Comment by jin-s13** [comment]
HRNET is a top-down model, which means the pose model assumes human bounding boxes are provided (using a separate human bbox detector). So we do not train bbox detection part in top-down pose models.

**Comment by LjIA26** [comment]
Thanks for your answer. That makes sense with the paper but how is the model localizing the objects ? I am using the animal pose hrnet model.

How does the model get this first part ? Is it only possible when one loads the pretrained weights?

**Comment by jin-s13** [comment]
I think you need to train an animal detector (maybe using mmdetection)

**Comment by jin-s13** [comment]
https://github.com/open-mmlab/mmpose/blob/master/demo/docs/2d_animal_demo.md#using-mmdetection-to-detect-animals

**Comment by LjIA26** [comment]
So, I am confused. The pretrained weights are for predicting the bounding boxes? Or the bounding boxes are never part of the prediction and it is just the bbox ground truth that has been used.

**Comment by jin-s13** [comment]
We use mmdet to predict bounding boxes, and use mmpose to predict pose.

**Comment by jin-s13** [comment]
So there are two models in total.
